images.yaml

```diff
@@ -641,6 +641,16 @@
 - name: quay.io/calico/cni
   patterns:
   - pattern: '>= v3.9.5'
+- name: quay.io/calico/ctl
```

Comment:
Maybe override the repo name, since this will just make it `giantswarm/ctl`

Reply:
Oops, yes, I meant to do that. Thanks.